generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs

```diff
@@ -134,8 +206,25 @@ export class <%= entityAngularName %>UpdateComponent implements OnInit {
         return this.dataUtils.openFile(contentType, field);
     }
 
-    setFileData(event, entity, field, isImage) {
-        this.dataUtils.setFileData(event, entity, field, isImage);
+    setFileData(event, field: string, isImage) {
```

Comment:
Why this change?

Reply:
Because we don't have entity at this stage: I'll first try to make it work in a generated project, then contribute to ng-jhipster with a new method.